acai-ios/MenuItem.swift

```diff
@@ -0,0 +1,146 @@
+//
+//  BowlItem.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 3/15/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import Foundation
+import IGListKit
+
+class MenuItem: ListDiffable {
+    
+    var title: String = ""
+    var price: Double = 0
+    var image: UIImage = UIImage()
+    var customizationOptions: [CustomizationOption] = []
+    
+    init (title: String, price: Double, customizationOptions: [CustomizationOption], image: UIImage) {
+        self.title = title
+        self.price = price
+        self.customizationOptions = customizationOptions
+        self.image = image
+    }
+    
+    func getSelectedOptionsPrice() -> Double {
+        var selectedOptionsPrice: Double = 0
+        for option in customizationOptions {
+            if (option.isSelected) {
+                selectedOptionsPrice += option.price
+            }
+        }
+        return selectedOptionsPrice
+    }
+    
+    func getSelectedOptions() -> [CustomizationOption] {
+        var selectedOptions: [CustomizationOption] = []
+        for option in customizationOptions {
+            if (option.isSelected) {
+                selectedOptions.append(option)
+            }
+        }
+        return selectedOptions
```

Comment:
you can do this in one line with `return customizationOptions.filter({$0.isSelected})`